pkg/status/cel-evaluator.go

```diff
@@ -37,7 +46,9 @@ type celEvaluator struct {
 func newCELEvaluator() (*celEvaluator, error) {
 	env, err := cel.NewEnv(
 		cel.Declarations(
-			decls.NewVar("obj", decls.NewMapType(decls.String, decls.Dyn)),
+			decls.NewVar(sourceObjectKey, decls.NewMapType(decls.String, decls.Dyn)),
+			decls.NewVar(statusKey, decls.NewMapType(decls.String, decls.Dyn)),
```

Comment:
I do not think that "returned.status" is a legitimate variable name.

Reply:
While I understand that the intention is to assign `returned` to whatever is returned from workstatuses, at the moment that is only `.status`. I thought of making this explicit. If you insist, I can make it implicit.

Reply:
My concern here is with CEL formalities. I do not think that "returned.status" is a legitimate variable name in CEL.

Reply:
I tested that it works, at a simple level at least. In all cases, changing in next push.